test/pagespecs/CalculatorTaxLiabilitiesPageSpec.scala

```diff
@@ -56,4 +60,78 @@ class CalculatorTaxLiabilitiesPageSpec extends ItSpec {
     taxLiabilitiesPage.clickOnStartNowButton()
     paymentTodayQuestionPage.assertPageIsDisplayed
   }
+
+  object LateDebitCase {
+    private val debit1Amount = 2500
+    private val debit2Amount = 2400
+    private val taxYearEnd = "2020-04-05"
+    private val lateDueDate = "2021-11-25"
+
+    val debit1: Debit = Debit(originCode = "IN1", debit1Amount, dueDate = lateDueDate, interest = None, taxYearEnd)
+    val debit2: Debit = Debit(originCode = "IN2", amount = debit2Amount, dueDate = lateDueDate, interest = None, taxYearEnd)
+
+    val taxpayerWithLateDebit: Taxpayer =
+      Taxpayer(
+        "Mr John Campbell",
+        List(address),
+        SelfAssessmentDetails(
+          saUtr,
+          communicationPreferences,
+          List(debit1, debit2),
+          List(
+            Return(taxYearEnd, issuedDate = "2019-11-10", lateDueDate, receivedDate = "2019-03-09"),
+            Return(taxYearEnd = "2018-04-05", issuedDate = "2017-02-15", lateDueDate, receivedDate = "2018-03-09"))))
+
+    object Expected {
+      object MainText {
+
+        def apply()(implicit language: Language): String = language match {
+          case English => mainTextEnglish
+          case Welsh   => mainTextWelsh
+        }
+
+        private val mainTextEnglish =
+          """Your Self Assessment tax bill is £4,900.00
+            |Self Assessment statement
+            |Due 25 November 2021
+            |First payment on account for tax year 2019 to 2020
+            |
+            |£2,500.00
+            |Due 25 November 2021
+            |Second payment on account for tax year 2019 to 2020
+            |
+            |£2,400.00
+            |Continue
+        """.stripMargin
+
+        private val mainTextWelsh =
+          """Crynodeb o’ch cyfrif Hunanasesiad £4,900.00
+            |Dadansoddiad Hunanasesiad
+            |Due 25 November 2021
+            |Taliad cyntaf ar gyfrif for tax year 2019 to 2020
+            |
+            |£2,500.00
+            |Due 25 November 2021
+            |Ail daliad ar gyfrif for tax year 2019 to 2020
+            |
+            |£2,400.00
+            |Yn eich blaen
+        """.stripMargin
+      }
+    }
+  }
+
+  "render late debit" in {
```

Comment:
New test case to capture a due date that happens to fall in a tax year following the tax year of the debit